doc/source/whatsnew/v0.25.0.rst

```diff
@@ -283,6 +283,7 @@ Timezones
 - Bug in :func:`Timestamp.tz_localize` and :func:`Timestamp.tz_convert` does not propagate ``freq`` (:issue:`25241`)
 - Bug in :func:`Series.at` where setting :class:`Timestamp` with timezone raises ``TypeError`` (:issue:`25506`)
 - Bug in :func:`DataFrame.update` when updating with timezone aware data would return timezone naive data (:issue:`25807`)
+- Bug in :func:`Series.apply` failed when the series is a timezone aware datetime index (:issue:`25959`)
```

Comment:
move to reshaping

instead of: timezone aware datetime index 
``timezone aware DatetimeIndex``, with double-backticks around DTI

Reply:
Fixed!